lib/battle/CBattleInfoCallback.cpp

```diff
@@ -855,6 +855,24 @@ AccessibilityInfo CBattleInfoCallback::getAccesibility() const
 		ret[BattleHex(0, y)] = EAccessibility::SIDE_COLUMN;
 	}
 
+	//special battlefields with logically unavailable tiles
+	std::vector<BattleHex> impassableHexes;
+	if(battleGetBattlefieldType().num == BFieldType::SHIP_TO_SHIP)
+		impassableHexes = 
+		{ 

```

Comment:
I think we shouldn't allow multi-line code in else-if without brackets. E.g this is bad: 
```
if(a)
	//comment
	dosomething();
```
So I want brackets around:
```
if(a)
{
	var =
	{
		1, 2, 3, 4,
		5, 6, 7, 8
	}
}
```
This is not request for actual changes yet. What everyone else thinks?